routes/create_user.routes.js

```diff
@@ -0,0 +1,16 @@
+module.exports = function(app, db) {
+
+    app.post('/create_user', (req, res) => {
+        let user_session_cookie = Math.floor(Math.random()*Number.MAX_SAFE_INTEGER);
+        var sql = "INSERT INTO user (use_name, use_session_cookie) VALUES (?, ?)";
+        var params = [req.body.username, user_session_cookie];
+            db.run(sql, params, (err) => {
+                if (err) {
+                    res.status(400).json({ "error": err.message });
+                    return;
+                }
+                res.cookie("user", user_session_cookie, {expires: new Date(2147483647000)});
```

Comment:
I don't understand the number you've given in new Date(). Is this just a random number to specify when the data saved in the user session cookie will expire?

Reply:
Die gegebene Zahl ist die Anzahl an Millisekunden zwischen dem 1. Januar 1970 und irgendeinem Datum 2034 oder so. Das ist einfach nur ein Weg, wie man Dates erstellen kann. Ich wollte einfach nur verhindern, dass die Cookies expiren.